envbuilder.go

```diff
@@ -1209,22 +1006,23 @@ func (fs *osfsWithChmod) Chmod(name string, mode os.FileMode) error {
 	return os.Chmod(name, mode)
 }
 
-func findDevcontainerJSON(options Options) (string, string, error) {
+func findDevcontainerJSON(c *Config, fs billy.Filesystem, logger Logger) (string, string, error) {
```

Comment:
I would keep the name `Options` to reduce the blast radius of PR changes. Happy to see more comments on this.

Also, for simple structs (this is a simple struct), it should be `c Config` or (`options Options`)

Reply:
What defines if a struct is complex or simple?